postgres/datadog_checks/postgres/config.py

```diff
@@ -173,7 +168,7 @@ def __init__(self, instance, init_config, check):
         )
         self.baseline_metrics_expiry = self.statement_metrics_config.get('baseline_metrics_expiry', 300)
 
-    def _build_tags(self, custom_tags, agent_tags, propagate_agent_tags=True):
+    def _build_tags(self, custom_tags, propagate_agent_tags=True):
```

Comment:
Should we consider updating the default to False?

Reply:
we should remove the default